vignettes/output_options.Rmd.src

```diff
@@ -0,0 +1,109 @@
+---
+title: "Output options"
+subtitle: "Get different kinds of output from OpenCage"
+author: "Daniel Possenriede, Jesse Sadler, Maëlle Salmon"
+date: "`r Sys.Date()`"
+description: >
+  "`oc_forward()`/`oc_reverse()` return lists of various type, namely data frames, JSON, GeoJSON or URLs, depending on the `return` value you specify. The possible `return` values are `df_list`, `json_list`, `geojson_list` and `url_only`."
+output:
+  rmarkdown::html_vignette:
+    df_print: kable
+vignette: >
+  %\VignetteIndexEntry{Output options}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, echo=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library("opencage")
+
+```
+
+{opencage} contains two types of main functions.
+
+1. The `oc_forward_df()`/`oc_reverse_df()` functions, which always return a single tibble.
+2. The `oc_forward()`/`oc_reverse()` functions, which return lists of various types, depending on the `return` value you specify. The possible `return` values are `df_list`, `json_list`, `geojson_list` and `url_only`.
+
+We have shown the `oc_forward_df()`/`oc_reverse_df()` functions in the [Introduction to opencage](intro_opencage.html) and the [Customise your query](customise_query.html) vignettes.
+The function arguments mentioned in the other vignettes are also available with `oc_forward()`/`oc_reverse()`, in general.
+Here we will show the different return values available with `oc_forward()`/`oc_reverse()`.
+
+## `df_list`
+
+The default return value is `df_list`.
+It returns a list of tibbles.
+
+```{r df_list}
+oc_forward("Casey Station", return = "df_list")
```

Comment:
It might be easier to show the difference with `oc_forward_df()` if there are two locations here. I know that this breaks the symmetry with the below examples, but I think it would be helpful.